thorlcr/activities/diskread/thdiskreadslave.cpp

```diff
@@ -411,6 +411,7 @@ void CDiskRecordPartHandler::close(CRC32 &fileCRC)
     if (partStream)
     {
         mergeStats(fileStats, partStream);
+        activity.mergeSubFileStats(partDesc, partStream);
```

Comment:
this means I think, that we have full disk read stats for the whole super (from prev. line), plus stats for each sub.

That's ok/good, but am wondering if there are supers with dozens of subs, is it getting to expensive to serialize detailed stats on every sub, especially as at the moment as only the StNumDiskReads is going to be used.

@ghalliday - what do you think?

Or if we are going to use them, should the other stats in the collection be preserved per sub, and published to the activity too?

Reply:
I've checked and I don't think the overhead will be that large.  It only serializes stats that have been filled in, and also packs the values so they are likely to only take a few bytes each.
I don't think there is an advantage to publishing the individual file stats to the activity.  Possibly to the workunit summary, but even then I doubt it.